htconfig.php

```diff
@@ -21,6 +21,15 @@
 $db_pass = 'mysqlpassword';
 $db_data = 'mysqldatabasename';
 
+// Set this variable to true if you want to use environment variables for mysql
+$db_use_env_vars = false; 
+if ($db_use_env_vars) {
```

Comment:
Pleas test for environment variables values before overriding the existing ones instead of a manual switch.

Reply:
Should i only test for one, or do a check for all of them?